WMF Framework/HasText.swift

```diff
@@ -6,6 +6,16 @@ extension String {
     public var wmf_hasNonWhitespaceText: Bool {
         return wmf_hasText && !self.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty
     }
+
+    public static func wmf_maybeNonWhitespaceString(maybeString: String?) -> String? {
```

Comment:
I realize it's also in the existing method above, but I believe the `wmf_` prefixes aren't necessary with Swift extensions like they were with Obj-C categories due to namespacing. In this case you could `@objc(wmf_maybe...)`